README.md

```diff
@@ -14,96 +14,135 @@ external dependencies, editor integration, and quick feedback.
 
 lorri supports Linux and macOS.
 
-## Tutorial
+## Setup on NixOS or with `home-manager` on Linux
 
-You can find the **lorri tutorial** [in the `./example`
-directory](./example). After following this tutorial, you will have
-a working setup of `lorri`, `direnv`, and working basic editor
-integration into Emacs.
+If you are using [NixOS][nixos] or [`home-manager`][home-manager] on Linux and
+a Nixpkgs channel at least as recent as `nixos-19.09`, you can get started with
+lorri as follows. Otherwise see the next section, [Setup on other
+platforms](#setup-on-other-platforms).
 
-## Support & Questions
+1. **Enable the daemon service.** Set `services.lorri.enable = true;` in your
+   NixOS [`configuration.nix`][nixos-service] or your home-manager
+   [`home.nix`][home-manager-service].
 
-Please use the [issue tracker](https://github.com/target/lorri/issues)
-for any problems or bugs you encounter. We are on `#lorri` on
-`freenode` ([Webchat][]), though we might not be responsive at all
-times.
+   This will automatically install both the `lorri` command and `direnv`
+   (both required for the next steps).
 
-[Webchat]: https://kiwiirc.com/nextclient/#irc://irc.freenode.net:+6697/#lorri
+2. **Set up the direnv hook for your shell.** See [this section][direnv-hook]
+   of the direnv documentation. Note that at this point, we have only tested
+   lorri with Bash. Please open a feature request if you want to use lorri from
+   a shell other than Bash and are experiencing issues.
```

Comment:
I added this after reading https://direnv.net/docs/hook.html#fish. The syntax specifically for `fish` looks very different there:

```fish
>direnv hook fish | source
```

I concluded that there is a separate set of instructions on that page because the Bash version wouldn't work in other shells. So I'm surprised that `eval "$(lorri direnv)"` works for you. But if it does, great! I'm happy to remove this caveat.

Reply:
@curiousleo the shell integration works on another level. The `.envrc` is always evaluated in a bash sub-process. Then direnv makes a diff of both environments and re-exports the diff to the target shell.

Reply:
> The `.envrc` is always evaluated in a bash sub-process.

Nice, thanks for the explanation!